llvm/test/CodeGen/AArch64/wineh-pac.ll

```diff
@@ -1,63 +1,65 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 6
 ; RUN: llc < %s -mtriple=aarch64-windows | FileCheck %s
 
 define dso_local i32 @func(ptr %g, i32 %a) "sign-return-address"="non-leaf" "sign-return-address-key"="b_key" {
+; CHECK-LABEL: func:
```

Comment:
Nit: As this moves around the test patterns, it's hard to see how this changes the output. Can you precommit a test update that just autoregenerates the patterns with the old code, so this just is a small diff for the changed codegen?